src/mots/cli.py

```diff
@@ -408,6 +408,10 @@ def create_parser(subcommand=None):
     parsers["clean"].add_argument(*path_flags, **path_args)
     parsers["check-hashes"].add_argument(*path_flags, **path_args)
 
+    parsers["clean"].add_argument(
+        "--refresh", action="store_true", help="refresh person data from Bugzilla"
```

Comment:
We have a `user` command, maybe we should refer to the data as `user data` instead of `person data`?